packages/kit/src/runtime/server/page/render.js

```diff
@@ -174,6 +174,29 @@ export async function render_response({
 	/** @param {string} path */
 	const prefixed = (path) => (path.startsWith('/') ? path : `${assets}/${path}`);
 
+	const serialized = { data: '', errors: 'null' };
+
+	try {
+		serialized.data = devalue(branch.map(({ server_data }) => server_data));
+	} catch (e) {
+		// TODO if we wanted to get super fancy we could track down the origin of the `load`
+		// function, but it would mean passing more stuff around than we currently do
+		const error = /** @type {any} */ (e);
```

Comment:
Maybe a small comment what this error throws (I'm guessing something like "X cannot be devalued")

Reply:
3e3d05de1468f6541154c7f284d49649016d7041